core/sagas/composes.js

```diff
@@ -115,8 +115,25 @@ function* fetchQueue() {
 
 function* fetchComposeTypes() {
   try {
-    const response = yield call(composer.getComposeTypes);
-    yield put(fetchingComposeTypesSucceeded(response));
+    const imageTypes = yield call(composer.getComposeTypes);
+    const imageTypeLabels = {
+      alibaba: "Alibaba Cloud (.qcow2)",
+      ami: "Amazon Web Services (.raw)",
+      "fedora-iot-commit": "Fedora IoT Commit (.tar)",
+      google: "Google Cloud Platform (.vhd)",
+      "hyper-v": "Hyper-V (.vhd)",
+      "live-iso": "Installer, suitable for USB and DVD (.iso)",
+      openstack: "OpenStack (.qcow2)",
+      "partitioned-disk": "Disk Image (.img)",
+      qcow2: "QEMU Image (.qcow2)",
+      "rhel-edge-commit": "RHEL for Edge Commit (.tar)",
+      vhd: "Azure Disk Image (.vhd)",
```

Comment:
Can you change "Azure Disk Image" to "Microsoft Azure". The others look good!

Reply:
This is fixed.